spec/lib/convenient_service/support/dependency_container/import_spec.rb

```diff
@@ -52,53 +52,19 @@
           .with_arguments(scope: scope)
       end
 
-      context "when `mod` does NOT include `ConvenientService::Support::DependencyContainer::Export`" do
-        let(:container) { Module.new }
-
-        let(:error_message) do
-          <<~TEXT
-            Module `#{container}` can NOT export methods.
-
-            Did you forget to include `ConvenientService::Container.export` into it?
-          TEXT
-        end
+      specify do
+        expect { import }
+          .to delegate_to(ConvenientService::Support::DependencyContainer::Commands::AssertValidContainer, :call)
+          .with_arguments(from: container)
+      end
 
-        it "raises `ConvenientService::Support::DependencyContainer::Errors::NotExportableModule`" do
-          expect { import }
-            .to raise_error(ConvenientService::Support::DependencyContainer::Errors::NotExportableModule)
-            .with_message(error_message)
-        end
+      specify do
+        expect { import }
+          .to delegate_to(ConvenientService::Support::DependencyContainer::Commands::AssertValidMethod, :call)
+          .with_arguments(method: method, full_name: full_name, scope: scope, from: container)
       end
 
       context "when `mod` includes `ConvenientService::Support::DependencyContainer::Export`" do
```

Comment:
Please, remove the whole context since its logic is tested by commands.

```ruby
specify do
  expect { import }
    .to delegate_to(ConvenientService::Support::DependencyContainer::Commands::ImportMethod, :call)
    .with_arguments(importing_module: user, exported_method: method, prepend: prepend)
    .and_return_its_value
end
```